README.md

```diff
@@ -23,7 +23,11 @@ custom:
       - production
       - staging
 
-    dashboards: true
+    dashboards: 
+      type: 
```

Comment:
It can be added additional check when getDashboardTemplates is called.
 
If dashboards is boolean(or string/string array) not an object, then return the form it with default properties
If it is object get type and based on that type return default or vertical.

With this approach is compatible with 
dashboards: true, 
dashboards: 'default', 
dashboards: 
    type: default

What do you think?